kafka-rest/src/test/java/io/confluent/kafkarest/integration/ClusterTestHarness.java

```diff
@@ -239,6 +237,25 @@ public void setUp() throws Exception {
     restServer.start();
   }
 
+  private void startBrokersConcurrently(int numBrokers) {
+    configs =
+        IntStream.range(0, numBrokers)
+            .mapToObj(brokerId -> KafkaConfig.fromProps(getBrokerProperties(brokerId)))
```

Comment:
Should be `KafkaConfig.fromProps(overrideBrokerProperties(brokerId, getBrokerProperties(brokerId)))`. I think that's why the tests are failing.

Reply:
Added call to overrideBrokerProperties.